src/components/filterSideBar/filterSideBar.jsx

```diff
@@ -0,0 +1,48 @@
+import {useProducts} from '../../context/productsContext'
+
+function FilterSideBar(){
+
+    const { data:{sortByPrice, categoryFilters, typeFilters, ratingFilter}, productsDispatch } = useProducts()
+    const categories = ['Dogs', 'Cats', 'Birds', 'Fishes', 'Bunnies']
+    const productType = ['Accessory', 'Food', 'Medicine', 'Treat', 'Toy']

```

Comment:
suggestion : you can separate out these arrays from this file and put in the data folder